app/models/annotations/dynamic.rb

```diff
@@ -195,7 +195,7 @@ def update_attribution
 
   def attribution_contains_only_team_members
     unless self.set_attribution.blank?
-      team_id = self.annotated.project ? self.annotated.project.team_id : self.annotated.team_id
+      team_id = self.annotated.respond_to?(:project) ? self.annotated.project.team_id : self.annotated.team_id
```

Comment:
I would prefer making sure we bypass project completely to get to the team - let's discuss

Reply:
Done